src/components/Popup.vue

```diff
@@ -14,15 +14,15 @@
         >
             <div
                 class="popup__close"
-                @click.prevent="close"
+                @click.prevent="onClose"
```

Comment:
wtf ? preco si to zmenil ved pred tym to robilo nieco ine ? 

close - emitlo aj `close` event aj zavolalo `closePopup` co neviem co robi ale asi nieco dolezite
onClose - len hide-ne popup, nezavola sa ani event ... ?